### DISCUSSION TITLE
Suggestion for adding a file in the User_Setups folder

### DISCUSSION THREAD

**Discussion by JonRobert** [Bodmer/TFT_eSPI#2644] [General]
I just succeeded in getting my 1st TFT_LCD display running the demo sketches.   I hit a stumbling block likely because I missed a statement somewhere regarding how to setup the User_Setup.h

My humble suggestion:

User_Setup_Instructions.txt
```
To "customize" the library for a particular display you need to:

1) find the ..\TFT_eSPI\User_Setups\Setup##_xxxxxxx.h   (such as Setup3_ILI9163.h) that most closely fits your hardware.
2) modify it as requied for your hardware
3) copy the modified file as "Setup_User.h" into the library root folder.  i.e.  ....\Library\TFT_eSPI

```

**Comment by Bodmer** [comment]
See header here for a simpler approach:
https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setup_Select.h
